### PR TITLE
besttrace: init at 1.3.2

### PR DESCRIPTION
**Pull request by luhuaei**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/besttrace/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "besttrace";
+  version = "1.3.2";
+
+  src = fetchzip {
+    url = "https://cdn.ipip.net/17mon/besttrace4linux.zip";
```
Comment (NickCao) [inline pkgs/tools/networking/besttrace/default.nix:8]: This URL does not seem to be immutable, any alternative avialable?

File: pkgs/tools/networking/besttrace/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  '';
+
+  postFixup = ''
+    patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" $out/bin/besttrace
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.ipip.net/product/client.html#besttrace";
+    description = "IPIP.net 开发的加强版 traceroute，附带链路可视化";
+    license = "custom";
```
Comment (symphorien) [inline pkgs/tools/networking/besttrace/default.nix:26]:
```suggestion
    license = licenses.unfree;
```

File: pkgs/tools/networking/besttrace/default.nix (hunk 1 above)
Comment (luhuaei) [reply pkgs/tools/networking/besttrace/default.nix:8]: This URL is application  official proviced, not other URL.

Comment (NickCao) [reply pkgs/tools/networking/besttrace/default.nix:8]: This can cause troubles in the long run, but guess we have no choice now but to leave it as is.

Comment (luhuaei) [reply pkgs/tools/networking/besttrace/default.nix:8]: Can use other personal repo? like https://github.com/zhucaidan/BestTrace-Linux

Comment (NickCao) [reply pkgs/tools/networking/besttrace/default.nix:8]: Or maybe https://archive.org

Comment (symphorien) [reply pkgs/tools/networking/besttrace/default.nix:8]: it mainly depends on whether the license allows redistribution.

Comment (luhuaei) [reply pkgs/tools/networking/besttrace/default.nix:8]: It not license description, don't known whether allows redistribution.

Comment (luhuaei) [reply pkgs/tools/networking/besttrace/default.nix:8]: I will cancel this PR

**Pull request closed without merging**